### PR TITLE
Addtasks

### PR DESCRIPTION
**Pull request by PVSASWIGGY**
performed self learning tasks

### CODE REVIEW

File: app/src/main/java/objectorientation/Box.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package objectorientation;
+
+public class Box {
+    int length,width,height;
+    public int volume(){
+        return length*width*height;
```
Comment (kratu11) [inline app/src/main/java/objectorientation/Box.java:6]: we can also add the colour of the box 

public int colour() { int colour= blue; return colour; }

**Comment by arpit2810** [comment]
All the code implemented is fine and to the point.

**Review by kratu11 — COMMENTED** [review]
rest all codes are fine!!

File: app/src/main/java/objectorientation/Box.java (hunk 1 above)
Comment (PVSASWIGGY) [reply app/src/main/java/objectorientation/Box.java:6]: ok

**Pull request merged**